rts/Lua/library/Types.lua

```diff
@@ -13,6 +13,16 @@
 
 ---@alias float3 xyz
 
+---Cartesian quadruple (XYZW)
+---
+---@class xyzw
+---@field x number
+---@field y number
+---@field z number
+---@field w number
+
+---@alias float4 xyzw
```

Comment:
Not really needed, unless you plan to use `float4` in places you intend to use `xyzw`. IIRC float3 exists only because we have mixed references between `xyz` and `float3` (float3 as a name makes not sense anyway imo).